internal/pkg/mapper/transformation/load.go

```diff
@@ -60,6 +60,9 @@ func (m *transformationMapper) MapAfterRemoteLoad(recipe *model.RemoteLoadRecipe
 			code.ConfigId = config.Id
 			code.BlockIndex = block.Index
 			code.Index = codeIndex
+			for i, script := range code.Scripts {
+				code.Scripts[i] = strhelper.NormalizeScript(script)
```

Comment:
+ este jedna drobnost ... v transformacii, v API, mozu jednotlive `scripts` obsahovat na konci biele znaky ... pri nacitani z `local` ich odstranujeme (lebo ich moze doplnit IDE), .... ale to musime spravit aj pri `remote` datach, ... preto uzivatel moze zadat prazdne znaky na koniec aj v UI.